app/command_utils.cc.template

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/app/command_utils.h"
+
+#include <iterator>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "base/feature_list.h"
+#include "base/no_destructor.h"
+#include "base/ranges/algorithm.h"
+#include "base/strings/string_split.h"
+#include "base/strings/string_util.h"
+#include "brave/app/brave_command_ids.h"
+#include "brave/components/commands/common/features.h"
+#include "chrome/app/chrome_command_ids.h"
+
+#define ADD_UNTRANSLATED_COMMAND(name) \
+  {                                    \
+    IDC_##name, {                      \
+      GetName(#name)                   \
+    }                                  \
+  }
+
+namespace commands {
+namespace {
+
+std::string GetName(const std::string& raw_name) {
+  auto words = base::SplitString(raw_name, "_", base::TRIM_WHITESPACE,
+                                 base::SPLIT_WANT_NONEMPTY);
+  for (auto& word : words) {
+    if (word.size() == 1) {
+      continue;
+    }
+    word = word[0] + base::ToLowerASCII(word.substr(1));
+  }
+  return base::JoinString(words, " ");
+}
+
+const base::flat_map<int, std::string>& GetCommandInfo() {
+  static const base::NoDestructor<base::flat_map<int, std::string>>
```

Comment:
Generator approach allows us to do `GetName()` stuff right in the generator script and use `base::StringPiece` instead of `std::string`. This way we won't waste memory on `std::strings`, but instead use `char[]` that are stored in the r/o section of the binary directly.

Also you should use `base/containers/fixed_flat_map.h` here and just declare this map as a global var like so:
```
constexpr auto kCommandInfos = base::MakeFixedFlatMap<...>(...);
```

a follow up would be nice here, but not critical.